src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/KeyStatisticServiceTests.cs

```diff
@@ -346,6 +346,164 @@ await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatistics = context.KeyStatistics.ToList();
+            var keyStat = Assert.Single(keyStatistics);
+            var keyStatText = Assert.IsType<KeyStatisticText>(keyStat);
+
+            Assert.Equal("title", keyStatText.Title);
+            Assert.Equal("Over 9000!", keyStatText.Statistic);
+            Assert.Equal(release.Id, keyStatText.ReleaseId);
+            Assert.Equal("trend", keyStatText.Trend);
+            Assert.Equal("guidanceTitle", keyStatText.GuidanceTitle);
+            Assert.Equal("guidanceText", keyStatText.GuidanceText);
+            Assert.Equal(0, keyStatText.Order);
+            Assert.Equal(_userId, keyStatText.CreatedById);
+            Assert.Null(keyStatText.UpdatedById);
+            Assert.InRange(DateTime.UtcNow.Subtract(keyStatText.Created).Milliseconds, 0, 1500);
+            Assert.Null(keyStatText.Updated);
+        }
+    }
+
+    [Fact]
+    public async Task CreateKeyStatisticText_NoRelease()
+    {
+        var release = new Release();
+
+        var contextId = Guid.NewGuid().ToString();
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            await context.Releases.AddRangeAsync(release);
+            await context.SaveChangesAsync();
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatisticService = SetupKeyStatisticService(context);
+            var result = await keyStatisticService.CreateKeyStatisticText(
+                Guid.NewGuid(),
+                new KeyStatisticTextCreateRequest
+                {
+                    Title = "title",
+                    Statistic = "Over 9000!",
+                });
+
+            result.AssertNotFound();
+        }
+    }
+
+    [Fact]
+    public async Task CreateKeyStatisticText_Order()
+    {
+        var release = new Release
+        {
+            KeyStatistics = new List<KeyStatistic>
+            {
+                new KeyStatisticText { Order = 0 },
+                new KeyStatisticText { Order = 1 },
+                new KeyStatisticDataBlock { Order = 2 },
+            },
+        };
+
+        var contextId = Guid.NewGuid().ToString();
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            await context.Releases.AddRangeAsync(release);
+            await context.SaveChangesAsync();
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatisticService = SetupKeyStatisticService(context);
+
+            var result = await keyStatisticService.CreateKeyStatisticText(
+                release.Id,
+                new KeyStatisticTextCreateRequest
+                {
+                    Title = "title",
+                    Statistic = "Over 9000!",
+                });
+
+            var viewModel = result.AssertRight();
+
+
+            Assert.Equal("title", viewModel.Title);
```

Comment:
Probably don't need these other non-Order checks in this particular test method, for brevity.